src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml

```diff
@@ -540,43 +540,45 @@
-                                                            <Style TargetType="Rectangle">
-                                                                <Style.Triggers>
-                                                                    <DataTrigger Binding="{Binding Path=Model.IsDeprecated}" Value="True">
-                                                                        <Setter Property="Fill">
-                                                                            <Setter.Value>
-                                                                                <ImageBrush ImageSource="/DynamoCoreWpf;component/UI/Images/View details - disabled.png" />
-                                                                            </Setter.Value>
-                                                                        </Setter>
-                                                                    </DataTrigger>
-                                                                    <DataTrigger Binding="{Binding Path=Model.IsDeprecated}" Value="False">
-                                                                        <Setter Property="Fill">
-                                                                            <Setter.Value>
-                                                                                <ImageBrush ImageSource="/DynamoCoreWpf;component/UI/Images/View details.png" />
-                                                                            </Setter.Value>
-                                                                        </Setter>
-                                                                    </DataTrigger>
-                                                                </Style.Triggers>
-                                                            </Style>
-                                                        </Rectangle.Style>
-                                                    </Rectangle>
-                                                    <TextBlock Name="textBlock"
-                                                               Padding="4,4,4,3"
-                                                               HorizontalAlignment="Right"
-                                                               VerticalAlignment="Center"
-                                                               Cursor="Hand"
-                                                               DockPanel.Dock="Right"
-                                                               FontFamily="{StaticResource ArtifaktElementRegular}"
-                                                               FontSize="11"
-                                                               Text="{TemplateBinding Content}" />
-                                                </DockPanel>
+                                                <Grid x:Name="viewDetailsGrid">
```

Comment:
Hi @RobertGlobant20, is this just to include the dockpanel inside the grid?

Reply:
@filipeotero the objective is for putting the highlight orange rectangle over the Button, if you see first we have the Border and inside the DockPanel, then If I put the Rectangle inside the Border it will crash due that only one element is allowed, then that's why I'm adding the Grid, in this case it will put the Rectangle over the DockPanel but due that the Rectangle is transparent there is no problem. I think this solution need to be applied for each button in which we need to put the orange rectangle.

Reply:
Thanks @RobertGlobant20!